packages/wrangler/src/deployment-bundle/entry.ts

```diff
@@ -60,9 +58,9 @@ export async function getEntry(
 	if (args.script) {
 		paths = resolveEntryWithScript(args.script);
 	} else if (config.main !== undefined) {
-		paths = resolveEntryWithMain(config.main, config.configPath);
+		paths = resolveEntryWithMain(config.main, config.projectRoot);
 	} else if (entryPoint) {
-		paths = resolveEntryWithEntryPoint(entryPoint, config.configPath);
+		paths = resolveEntryWithEntryPoint(entryPoint, config.projectRoot);
```

Comment:
I'm not sure why this isn't a type error, but I think the return value here is used further down, and should probably be changed to `config.projectRoot` as well:

```ts
89: const projectRoot = paths.projectRoot ?? process.cwd();
```

Reply:
Oops I missed that one.
It is not a type error it is just redundant further down.